### ISSUE TITLE
Have a way to detect redundant # vulture: ignore lines

### ISSUE THREAD

**Issue by nth10sd** [Gatewatcher/pytest-vulture#21]
While we can ignore vulture errors by adding `# vulture: ignore`, when the function is subsequently used (thus rendering the ignore portion invalid), we should have a way to detect when the `# vulture: ignore` portion is no longer needed.

**Comment by MoranAbadieGatewatcher** [comment]
yes good idea, but it is not that simple, I will try to do this when I will have time :)

**Comment by MoranAbadieGatewatcher** [comment]
When you think about it, this functionality seems more appropriate as a feature within Vulture itself (https://github.com/jendrikseipp/vulture) :). While we initially added support for `# vulture: ignore` for our own purposes, handling unused code annotations feels more like Vulture's responsibility than that of a pytest plugin.
